fundamentals/impl.tex

```diff
@@ -454,15 +455,20 @@ \subsubsection{Cycamore}
 \textbf{Entity} & \textbf{Archetype} & \textbf{Functionality} \\
 \hline
 Facility & BatchReactor & A reactor model that handles batch refueling, based on pre-determined recipes of compositions. \\
-Facility & Source & This facility generates material of the composition and commodity type specified as input.  \\
-Facility & Sink & This facility is capable of accepting a finite or infinite quantity of some commodity produced in the simulation. \\
 Facility & EnrichmentFacility & This facility enriches uranium at a specified capacity. \\
+Facility & Fab$^*$ & This facility fabricates fuel material based on separated streams. \\
+Facility & Reactor$^*$ & This facility, based on BatchReactor, requests any number of input fuel types and transmutes them to static compositions upon discharge.\\
```

Comment:
Yesh, I know. I thought maybe having both there and noting which are in 1.0 rather than 1.3 is probably the most clarity we'll get until 1.3 is actually out. 
I'll make a note to fix this in the post-1.3 submission version.